scripts/zones/King_Ranperres_Tomb/mobs/Vrtra.lua

```diff
@@ -66,9 +66,7 @@ entity.onMobDeath = function(mob, player, optParams)
 end
 
 entity.onMobDespawn = function(mob)
-    -- Set Vrtra's spawnpoint and respawn time (3-5 days)
-    UpdateNMSpawnPoint(mob:getID())
-    mob:setRespawnTime(math.random(259200, 432000))
+    xi.mob.nmTODPersist(mob, math.random(259200, 432000)) -- 3 to 5 days
```

Comment:
I always thought this was in 30 min windows as well. I guess everything just says 3-5 days O.o